chpl/chpl-resources/src/main/resources-production/log4j2.xml

```diff
@@ -483,34 +406,10 @@
             additivity="false">
             <AppenderRef ref="updateListingStatusJob" />
         </Logger>
-        <Logger name="update2014ListingStatusJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="update2014ListingStatusJob" />
-        </Logger>
         <Logger name="remove2014UploadTemplateJobLogger" level="INFO"
             additivity="false">
             <AppenderRef ref="remove2014UploadTemplateJob" />
         </Logger>
-        <Logger name="removeV17UploadTemplateJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="removeV17UploadTemplateJob" />
-        </Logger>
-        <Logger name="addV18UploadTemplateJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="addV18UploadTemplateJob" />
-        </Logger>
-        <Logger name="addV19UploadTemplateJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="addV19UploadTemplateJob" />
-        </Logger>
-        <Logger name="createTestToolJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="createTestToolJob" />
-        </Logger>
-        <Logger name="removeCriteriaJobLogger" level="INFO"
-            additivity="false">
-            <AppenderRef ref="removeCriteriaJob" />
-        </Logger>
         <Logger name="modifyTestFunctionalityJobLogger" level="INFO"
```

Comment:
Does this job still exist?

Reply:
It gets removed in a different ticket OCD-3425